static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -245,6 +245,7 @@ export function WidgetInteractiveTitle({
       value={chartSetting}
       onChange={handleChange}
       triggerProps={{borderless: true, size: 'zero'}}
+      offset={4}
```

Comment:
Use smaller menu offset, otherwise we'll see part of the description text in between the trigger and the menu.
<img width="262" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/4d1a61ca-a58a-4da1-b0d1-5c17f8513cff">
<img width="262" alt="image" src="https://github.com/getsentry/sentry/assets/44172267/f4171ff9-0352-4020-a4ca-61a9039a8fb4">